ci/jobs/docs.yaml

```diff
@@ -48,9 +48,15 @@
 
             chmod 644 /home/jenkins/.ssh/known_hosts
 
+            # create a virtualenv in which to install packages needed to build docs
+            virtualenv -p /usr/bin/python3 --system-site-packages ~/docs_ve
```

Comment:
As of Python 3.3, the `venv` module is shipped with the standard library. The following commands both create a Python 3 virtualenv:

    python3 -m venv env
    virtualenv -p python3 env

One benefit to using the `venv` module shipped with the standard library is that one less dependency is needed.